src/utils/shared.ts

```diff
@@ -36,3 +36,19 @@ export const getTokenRegistryAddress = (document: WrappedOrSignedOpenAttestation
   const issuerAddress = utils.getIssuerAddress(document);
   return issuerAddress instanceof Array ? issuerAddress[0] : issuerAddress;
 };
+
+export const getChain = (rawDocument: WrappedOrSignedOpenAttestationDocument): string | undefined => {
+  const warn = () =>
+    console.warn(
```

Comment:
This part looks like something coming from #520? Are changes from that PR already merged into this one?

Reply:
nope, i'm planning to reuse #520's getChainId/getChain function, but we need to resolve https://github.com/TradeTrust/tradetrust-website/pull/520#issuecomment-1053992304 before deciding to either wholesale use getChainId or stick with getChain